core/src/test/java/kafka/test/annotation/ClusterTestDefaults.java

```diff
@@ -42,4 +42,5 @@
     boolean autoStart() default true;
     // Set default server properties for all @ClusterTest(s)
     ClusterConfigProperty[] serverProperties() default {};
+    ClusterConfigDisplayTag[] displayTags() default {};
```

Comment:
Could we add this to default annotation if we do have such use case?

Reply:
do you mean put the tags in the ClusterTestDefaults?